src/components/buttonMenu/buttonMenu.stories.svelte

```diff
@@ -28,6 +28,7 @@
 <Template let:args>
   <div class="container">
     <ButtonMenu>
+      <span slot="anchor-content"> Click Me! </span>
```

Comment:
Why the spaces around ` Clicke Me! `?

Reply:
I think the formatter did it - I originally had it split across a couple of lines - I removed them and its still happy though :smile: